src/components/auth/Login.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import * as Yup from 'yup';
+import { Formik } from 'formik';
+import {
+  Box,
+  Button,
+  Card,
+  CardContent,
+  Container,
+  TextField,
+  Typography,
+  makeStyles,
+} from '@material-ui/core';
+
+import './styles.css';
+
+const useStyles = makeStyles({
+  root: {
+    background: '#f1f1f1',
+    margin: 'auto',
+    width: '30rem',
+    height: '30rem',
+    position: 'absolute',
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+  },
+  box: {
+    margin: 'auto',
+    position: 'absolute',
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+  },
+  title: {
+    fontSize: 14,
+  },
+  pos: {
+    marginBottom: 12,
+  },
+});
+
+function Login() {
+  const classes = useStyles();
+
+  return (
+    <Card className={classes.root}>
+      <CardContent>
+        <Container className={classes.box}>
+          <Formik
+            initialValues={{
+              username: '',
+              password: '',
+            }}
+            validationSchema={Yup.object().shape({
+              username: Yup.string()
+                .max(255)
+                .required('Username is required'),
+              password: Yup.string()
+                .max(255)
+                .required('Password is required'),
+            })}
+            onSubmit={(data) => {
+              alert(JSON.stringify(data));
+            }}
+          >
+            {({
+              errors,
+              handleBlur,
+              handleChange,
+              handleSubmit,
+              isSubmitting,
+              touched,
+              values,
+            }) => (
+              <form onSubmit={handleSubmit}>
+                <Box mb={3}>
```

Comment:
What mb={3} means? Usually hard coded values are hard to understand from reading code perspective. A solution for that is creating a constant with a meaningful name and using here.